addons/mail/static/src/models/popover_view/popover_view.js

```diff
@@ -6,7 +6,7 @@ import { clear } from '@mail/model/model_field_command';
 
 registerModel({
     name: 'PopoverView',
-    identifyingFields: ['threadViewTopbarOwner', 'channelInvitationForm'],
+    identifyingFields: [['composerViewOwner', 'messageActionListOwner', 'threadViewTopbarOwner']],
```

Comment:
I think we should specify on the field name not only the owner, but also the purpose. Because nothing should prevent an owner for having different popover for different purposes.

Reply:
That was what I wanted to do, but I realised it's problematic with the expressiveness of `identifyingFields`.
We cannot properly do `(A and B) or (C and D) or (E and F)`.

But I think it's fine with just owners. Right now there's only 1 use of popover view per model owner. If we needed more popover views for a given model that owns it, we could just use different field names. The inverse precisely target the popover view in scope of feature, like `reactionPopoverView`.

Reply:
I meant just in the field name:

composerViewOwner -> composerViewOwnerAsReaction
messageActionListOwner -> messageActionListOwnerAsReaction
threadViewTopbarOwner -> threadViewTopbarOwnerAsInvite

The inverse name is helpful, but with the current name it's hard to understand the methods.

Example in the code below, it's hard to understand why would `composerViewOwner` imply `buttonEmojisRef`
```
            if (this.composerViewOwner) {
                return this.composerViewOwner.buttonEmojisRef;
            }
```

Reply:
good idea, will do this